### PR TITLE
refactored a bit of code, and made upvote-downvote exclusive

### PR DESCRIPTION
**Pull request by mitkoyanev**
refactored a bit of code, and made upvote-downvote exclusive to each other. Small PR

### CODE REVIEW

File: src/components/Stories/TrendingStoryDetail.js [hunk 1]
```diff
@@ -22,12 +22,33 @@ class TrendingStoryDetail extends Component {
...
   contentHtml(data) {
     return ReactHtmlParser(data);
   }
 
+  approveVote(response, story) {
+    var arr = this.props.TrendingStoriesReducer.approvedStories;
+    var indexOfId = _.findIndex(arr, { id: story.id });
+    arr.splice(indexOfId, 1, response.data);
+
+    let keyDown = story.id + 'up';
```
Comment (steventhan) [inline src/components/Stories/TrendingStoryDetail.js]: If the user can only either upvote or downvote a story, we should probably store the voted story id in a set in `localStorage` for quick lookup

Comment (sheelasathyanarayana) [reply src/components/Stories/TrendingStoryDetail.js]: But, in my opinion we should not prevent user from up or down voting. We need to decrease the count of the other vote when user clicks on one.

Comment (steventhan) [reply src/components/Stories/TrendingStoryDetail.js]: sure, i think site like reddit also does that

Comment (mitkoyanev) [reply src/components/Stories/TrendingStoryDetail.js]: I made the alteration to just 1 local variable. I, too, agree that the user should be able to change their mind fluidly. I propose we leave that for the last sprint, for touch-up

**Review by jhalaa — APPROVED** [review]

**Pull request merged**